apps/chat/channels.py

```diff
@@ -429,13 +429,20 @@ def _reset_session(self):
         self._create_new_experiment_session()
 
     def _create_new_experiment_session(self):
+        """Creates a new experiment session. If one already exists, the participant will be transfered to the new
+        session
+        """
+        if not self.experiment_session:
+            participant = Participant.objects.create(
```

Comment:
Shouldn't this use `get_or_create`?

Reply:
Good catch. I'll add a test for this case and update the code